.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,86 @@
+name: 🐛 Bug report
+description: Create a report to help us improve
+title: "[BUG] <title>"
+labels: ['bug, untriaged']
+body:
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Related component
+      description: Choose a specific OpenSearch component your bug belongs to.  If you are unsure which to select or if the component is not present, select "Other".
+      multiple: false
+      options:
+        - Other
+        - Build
+        - Clients
+        - Cluster Manager
+        - Extensions
+        - Indexing:Performance
+        - Indexing:Replication
+        - Indexing
+        - Libraries
+        - Plugins
+        - Search:Aggregations
+        - Search:Performance
+        - Search:Query Capabilities
+        - Search:Query Insights
+        - Search:Relevance
+        - Search:Remote Search
+        - Search:Resiliency
+        - Search:Searchable Snapshots
+        - Search
+        - Storage:Durability
+        - Storage:Performance
+        - Storage:Remote
+        - Storage:Snapshots
+        - Storage
+    validations:
+      required: true
+  - type: textarea
```

Comment:
These 4 bottom boxes take up a lot of web page space, could we consolidate them into a single text box "Additional Details" that has commented out <!-- --> instructions and hints like including host information or plugins?

Reply:
Consolidated this - can see my fork for what it looks like - https://github.com/mch2/OpenSearch/issues/new/choose